

#---(in meters)---
footprint: [[-0.15,-0.13],[0.15,-0.13],[0.15,0.13],[-0.15,0.13]]
transform_tolerance: 0.7
map_type: costmap
obstacle_layer:
 enabled: true
 obstacle_range: 3.0
 raytrace_range: 3.5
 inf_is_valid: true
 inflation_radius: 0.18
 max_obstacle_height:  2.0
 min_obstacle_height:  0.0
 observation_persistence: 0.0
 track_unknown_space: false
 combination_method: 1
#  clearing: true
 expected_update_rate: 0.0
 observation_sources: laser_scan_sensor
 laser_scan_sensor:
  {data_type: LaserScan,sensor_frame: laser, topic: scan, marking: true, clearing: true}


inflation_layer:
  enabled:              true
  cost_scaling_factor:  5.0  # exponential rate at which the obstacle cost drops off (default: 10)
  inflation_radius:     0.4  # max. distance from an obstacle at which costs are incurred for planning paths.


